mne/forward/_make_forward.py

```diff
@@ -352,6 +352,10 @@ def make_forward_solution(info, trans, src, bem, fname=None, meg=True,
     fwd : instance of Forward
         The forward solution.
 
+    See Also
+    --------
+    forward.do_forward_solution
```

Comment:
shouldn't need to prepend `forward.`

read_forward_solution, write_forward_solution